src/services/organizationService.ts

```diff
@@ -490,6 +495,104 @@ export async function registerOrganization(
   }
 }
 
+export function addDelegate(
+  ipaCode: string,
+  userEmail: string
+): TaskEither<
+  // tslint:disable-next-line:max-union-size
+  IResponseErrorInternal | IResponseErrorNotFound | IResponseErrorConflict,
+  IResponseSuccessRedirectToResource<Organization, Organization>
+> {
+  const genericErrorHandler = (error: unknown) => {
+    log.error("An error occurred adding the delegate. %s", error);
```

Comment:
not required for this pr but adding the function as a prefix is useful when logging (ie:
`log.error("organizationService#addDelegate|An error ...")`